React/main/src/app/components/App/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+import Game from '~screens/Game';
+
+class App extends Component {
```

Comment:
When the Component is stateless we write it like a function. For example:

function App() {
    return...
}